project/CommonSettings.scala

```diff
@@ -183,6 +183,8 @@ object CommonSettings {
       //the default provided by sbt native packager
       //which is 'demiourgos728'
       Docker / daemonUser := "bitcoin-s",
+      //needed for umbrel to run
+      Docker / daemonUserUid := Some("1000"),
```

Comment:
Maybe make this an enviroment variable, so can be configured for different node packagers

Reply:
Hmm, not a bad idea. is there security implications here? cc @rorp